### PR TITLE
Add accounts state

### PR DESCRIPTION
**Pull request by RiccardoM**
## Description
This PR adds the accounts state for the genesis of the first Desmos mainnet.  
It also includes a Python script used to get generate this data from a CSV file.

### CODE REVIEW

**Review by kwunyeung — COMMENTED** [review]
LGTM for generating vesting accounts. I wonder how the bank balances will be handled. Also I suggest we should have a few, 10 DSM?, in balances so that everyone can pay tx fee for transactions after genesis.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+
+      - name: Setup Desmos 💿
+        run: |
+          curl https://github.com/desmos-labs/desmos/releases/download/v1.0.0/desmos-1.0.0-linux-amd64 > desmos
+          chmod +x ./desmos
+          ./desmos init test
+
+      - name: Check genesis validaty ✅
```
Comment (bragaz) [inline .github/workflows/test.yml:39]:
```suggestion
      - name: Check genesis validity ✅
```

**Review by bragaz — COMMENTED** [review]
Looks good to me too

**Comment by RiccardoM** [comment]
Ok I've also added some workflows to make sure that: 

1. Genesis transactions do not have more than 10.000 DSM as self delegated amount
2. The overall amount of tokens will be 100.000.000 DSM.

**Review by dadamu — APPROVED** [review]
LGTM

File: scripts/check-gentx.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+
+def check_self_delegation_validity(self_delegate: Coin):
+    """
+    Checks if the given self delegate amount is valid.
+    An amount is considered valid only if it does not exceed 10.000 DSM.
+
+    :param self_delegate: Self delegation amount to be checked.
+    """
+    if self_delegate.amount > 10_000_000_000:
```
Comment (bragaz) [inline scripts/check-gentx.py:48]: maybe I'm wrong but, aren't this too many zeros? Shouldn't be: `10_000_000`?

Comment (RiccardoM) [reply scripts/check-gentx.py:48]: > maybe I'm wrong but, aren't this too many zeros? Shouldn't be: `10_000_000`?

The amounts are always expressed in `udsm`. 10.000.000 (10 mln) DSM are equals to 10.000.000.000.000 udsm as 1 DSM = 1.000.000 udsm

Comment (bragaz) [reply scripts/check-gentx.py:48]: Oh right, thanks for the clarification 👍

**Review by bragaz — APPROVED** [review]

**Pull request merged**